invenio_app_ils/config.py

```diff
@@ -207,6 +207,7 @@ def _(x):
 
 ACCOUNTS_REST_CONFIRM_EMAIL_ENDPOINT = "/accounts/confirm-email"
 
+SECURITY_POST_LOGOUT_VIEW = "/logged_out"
```

Comment:
In the same spirit of the comment below, I would set this to `/logged_out` only if `dev` instance.
To be able to do that, you will have to move it to the `ext.py` on app initialization. This is because you need the `current_app` context, which you don't have in the config file when parsed (the app is not running).

I would also add a nice comment explaining why it is needed. WDYT?